records_mover/db/vertica/records_import_options.py

```diff
@@ -48,14 +48,18 @@ def vertica_import_options(unhandled_hints: Set[str],
     quiet_remove(unhandled_hints, 'quotechar')
     quiet_remove(unhandled_hints, 'quoting')
 
+    # TODO: Does this perhaps support more things on input for all of
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker

Reply:
Moved to Asana